site/data/qasida/safati-al-nazra.json

```diff
@@ -0,0 +1,136 @@
+{
+  "name": {
+    "ara": "صَفَتِ النَّظْرَةُ",
+    "en": "Ṣafati al-Naẓra - Clear was the Vision"
```

Comment:
we also add the transliteration of the title here for search engines to find us better